<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>671</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Opérations sur les données</string>
   </property>
   <widget class="QPushButton" name="loading_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger des données</string>
    </property>
   </widget>
   <widget class="QPushButton" name="missing_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeurs manquantes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outlier_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeurs aberrantes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicates_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Doublons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sampling_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Echantillonnage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desc_analysis_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyses descriptives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stat_test_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tests statistiques</string>
    </property>
   </widget>
   <widget class="QPushButton" name="datavis_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualisation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="data_table_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les données</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_res_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportation de resultats</string>
    </property>
   </widget>
   <widget class="QPushButton" name="correct_na_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Correction données manquantes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_data_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportation de données</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="quitter">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom du projet : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ope">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Opérations éffectuées</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
